pkg/multiTenancyPlugins/authorization/defaultAuthZImpl.go

```diff
@@ -41,6 +41,12 @@ func (defaultauthZ *DefaultAuthZImpl) Handle(command string, cluster cluster.Clu
 			if err := json.NewDecoder(bytes.NewReader(reqBody)).Decode(&containerConfig); err != nil {
 				return err
 			}
+			//Disallow a user to create the special labels we inject : headers.TenancyLabel
+			res := strings.Contains(string(reqBody), headers.TenancyLabel)
```

Comment:
Is it enough to just search for the key somewhere or you need to search for --label pattern?